src/content/resources/faq.md

```diff
@@ -357,6 +356,16 @@ developers to access new mobile OS features and capabilities
 immediately. Developers don't have to wait for the Flutter team
 to expose the new mobile OS capability.
 
+### Does Flutter support code push?
+
+Code push, or the ability to push app updates directly to a user's device,
+isn't directly supported by Flutter. However, there is a third
+party solution, called [Shorebird][]. For more information,
+check out the [shorebird_code_push][] package on pub.dev.
```

Comment:
I suspect their homepage is sufficient for folks who want to learn more. All things being equal, the less links we need to keep updated the better so consider removing the second sentence.